--- qutip/cy/graph_utils.pyx.old    2017-03-14 15:12:18.535506159 +0530
+++ qutip/cy/graph_utils.pyx        2017-03-14 16:43:49.107950996 +0530
@@ -52,6 +52,7 @@
     int idx
 
 ctypedef _int_pair int_pair
+ctypedef int (*cfptr)(int_pair, int_pair)
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
@@ -68,8 +69,8 @@
     for kk in range(nrows):
         pairs[kk].data = x[kk]
         pairs[kk].idx = kk
-    
-    sort(pairs.begin(),pairs.end(),int_sort)
+    cdef cfptr cfptr_ = &int_sort
+    sort(pairs.begin(),pairs.end(),cfptr_)
     cdef int * out = <int *>PyDataMem_NEW(nrows *sizeof(int))
     for kk in range(nrows):
         out[kk] = pairs[kk].idx

